conrad-blucher-institute/semaphore
# 444 meta table for semaphore failing

To test you can run test:
`docker exec semaphore-core python3 -m pytest`

Try running a model:
`docker exec semaphore-core python3 src/semaphoreRunner.py -d TestModels/test_dspec.json`

You can also cause semaphore to fail somewhere.

You can see the results by querying the db:
SELECT * FROM outputs
JOIN model_runs ON outputs."id" = model_runs."outputID"

This PR contains a DB migration script, the number might need to be changed on merge
